README.md

```diff
@@ -19,74 +19,94 @@ These are related projects that are either already setup and available for use o
 
-Use the AWS console or command line tool to create a host. For example:
+The following specification has worked well for a small-scale production environment :
 
 * Ubuntu Server 16.04
 * r4.xlarge
 * 250GB disk
 
-We will refer to this as the host VM throughout the rest of the documentation. It will run the Docker containers for all of the components listed below.
+For development work the following specifications have worked well in the past:
+* Ubuntu Server 16.04
+* m5.large
+* 60 GB disk
+
 
-You should make a note of your security group name and ID and ensure you can connect via ssh.
+#### Configuring the ports in your VM
+Open inbound ports on your security group. Use the table below as a guide. Make sure you add /32 to the *Elastic IP*.
 
-**Note** We have had problems when uploading big files to Virginia (~25GB). If possible, set up your AWS anywhere else but Virginia.
+| Type | Port | Source | Description |
+| --- | --- | --- | --- |
+| HTTP | 80 | 0.0.0.0/0 | |
+| HTTP | 80 | ::/0 | |
+| HTTPS | 443 | 0.0.0.0/0 | |
+| HTTPS | 443 | ::/0 | |
+| All TCP | 0 - 65535 | _Your VM's Elastic IP_ | |
+| All TCP | 0 - 65535 | _Your Security Group ID_ | | 
+| Custom TCP Rule | 9000 | _Your VM's Elastic IP_ | webservice port |
+| Custom TCP Rule | 9200 | _Your VM's Elastic IP_ | Elasticsearch |
+| SSH | 22 | 0.0.0.0/0 | |
 
-### AWS Tasks
 
-Make sure you do the following:
+#### <a name="makeip"></a>Create and assign an _Elastic IP_ for your VM 
+1. Go to _AWS console_, _Compute_,  _EC2 Dashboard_. There click *running instances*. 
+2. Find your VM and make it active by clicking anywhere in that line.
+3. Under the drop-down *Actions*, go to *Networking*, *Manage IP Addreses*.
+4. Click on *Allocate an Elastic IP*. It will automatically create an IP address and show it on the screen.
+5. Write down that _Elastic IP_. At this point, whatever IP address is shown under IPv4 Public IP for your VM is not the current instance's IP address. Next you need to associate the _Elastic IP_ with your VM.
+6. Back in _EC2 Dashboard_ go to *Elastic IPs*. The IP address just created should be in that list. Check it and under *Actions*, *Associate*, choose resource type "Instance", and choose your EC2 (e.g., searching by its name).
+7.  <a name="sshconnect"></a>In _EC2 Dashboard_ make your VM active by clicking it. Then click _Connect_ on top. The example in that window shows you how to ssh into your VM from a terminal.
 
-* assign an Elastic IP (a static IP address) to your instance
-* open inbound ports on your security group
-    * 80 <- world
-    * 22 <- world
-    * 443 <- world
-    * all TCP <- the elastic IP of the VM (Make sure you add /32 to the Elastic IP)
-    * all TCP <- the security group itself
 
-#### Adding private SSH key to your VM
+#### Adding a private SSH key to your VM
```

Comment:
I was just reading up on this, and a .pem is not an ssh key. It is a certificate.

So maybe your title should be something like `SSHing to your VM`.

Reply:
Good catch. I read more about it here: https://stackoverflow.com/questions/6394762/how-to-setup-ssh-access-for-amazon-ec2-instance. Amazon never calls it a certificate but a key pair so I used that expression here and linked to the appropriate AWS website.